client/src/lifecycleManager.js

```diff
@@ -140,17 +148,21 @@ class LifecycleManager extends LuigiClientBase {
   }
 
   _tpcCheck() {
-    if (this.currentContext?.internal?.thirdPartyCookieCheck?.disabled || this.disableTpcCheck) {
+    const tpcCheckDisabled =
+      this._isTpcCheckDisabled() || this.currentContext?.internal?.thirdPartyCookieCheck?.disabled;
```

Comment:
moving the check
`this.currentContext?.internal?.thirdPartyCookieCheck?.disabled;`
into `_isTpcCheckDisabled()` could save one LOC ;)